cmd/nomos/status/client.go

```diff
@@ -168,6 +168,9 @@ func (c *ClusterClient) clusterStatus(ctx context.Context, cluster, namespace st
 
 	if namespace != "" {
 		c.namespaceRepoClusterStatus(ctx, cs, namespace)
+		if namespace == "config-management-system" {
+			c.multiRepoClusterStatus(ctx, cs)
```

Comment:
The function list both RootSyncs and RepoSyncs, but you only need RootSyncs when namespace is cms.

Reply:
the function has been split to do root only for c-m-s namespace